pytest_tests/pytest.ini

```diff
@@ -1,6 +1,6 @@
 [pytest]
 log_cli = 1
-log_cli_level = DEBUG
+log_cli_level = ERROR
```

Comment:
How about INFO of some kind? Is ERROR enough to untangle test failures?

Reply:
> How about INFO of some kind? Is ERROR enough to untangle test failures?

First, in the report will be the test results:
https://nspcc-dev.github.io/neofs-node/95/

If the test falls, the full error log will be available:
https://github.com/nspcc-dev/neofs-node/actions/runs/5025178563/jobs/9011808006

![image](https://github.com/nspcc-dev/neofs-testcases/assets/126654550/4ca0609d-d7c9-4efd-ad0c-07f6d6f225a5)

Reply:
OK, let's try it this way.